client/jetpack-connect/jetpack-connect-notices.jsx

```diff
@@ -110,6 +114,12 @@ export class JetpackConnectNotices extends Component {
 		}
 
 		switch ( noticeType ) {
+			case ACTIVATE_FAILURE:
+				noticeValues.text = translate(
+					'We were unable to activate Jetpack. Please upgrade your WordPress version.'
```

Comment:
We might want to be more specific here, "... Please upgrade to the latest version of WordPress. Jetpack needs version x.x or higher.", and add the minimum version as a constant in `client/jetpack-connect/constants.js`?